src/components/KeyboardShortcuts/index.tsx

```diff
@@ -215,6 +263,7 @@ const mapStateToProps = (
 
   return {
     currentAnchor: location.hash.replace(/^#/, ''),
+    messageUid: typeof messageUid === 'string' ? messageUid : '',
```

Comment:
Oh, `queryString` returns an array if `messageUid` appears more than once on the query string. That's annoying. I guess this will ignore multiple query string values but that's fine.